File: C&C++/Purvansh Rathore/Day 5 Q 2.c
//DAY 5

//Question 2
#include <stdio.h>

// Function to find the GCD (Greatest Common Divisor) of two numbers
int gcd(int a, int b) {
    while (b != 0) {
        int temp = b;
        b = a % b;
        a = temp;
    }
    return a;
}

int main() {
    // Take input from the user
    int num1, num2;
    printf("Enter the first number: ");
    scanf("%d", &num1);
    printf("Enter the second number: ");
    scanf("%d", &num2);

    // Find the LCM (Least Common Multiple) using the GCD
    int lcm = (num1 * num2) / gcd(num1, num2);

    printf("The LCM of %d and %d is: %d\n", num1, num2, lcm);

    return 0;
}

File: Python/Anshika Srivastava/day5q2.py
a=int(input("Enter first number "))
b=int(input("Enter second number "))
i=a
d=0
while(i<=b):
    j=2
    c=0
    while(j<i):
        if(i%j==0):
            c=1
            break
        j=j+1
            
    if(c==0):
        print(i)
        d=1
    i=i+1
if(d==0):
    print("No prime numbers in entered range")
